.github/workflows/go.yml

```
@@ -0,0 +1,49 @@
+        go build -v -o bin/ ./...
+
+    - name: Upload build artifacts
+      uses: actions/upload-artifact@v3
+      with:
+        name: binaries
+        path: bin/
+
+    - name: Test
+      run: |
+        go test -v ./... -json | tee test-results.json
+        
```

yamllint

[error] trailing spaces

(trailing-spaces)